agency/README.md

```diff
@@ -209,7 +209,7 @@ Body Params:
 
 | Field     | Type                           | Field Description                                                                                       |
 | --------- | ------------------------------ | ------------------------------------------------------------------------------------------------------- |
-| `result`  | String                         | Responds with number of successfully written telemetry data points and total number of provided points. |
+| `result`  | String                         | Responds with number of successfully written telemetry data points and total number of provided points. Format to use: `{success}/{total}`, both amounts as integers. |
```

Comment:
How about adding `successes` and `total` as integers, in addition to a `result` string, or dropping `result` entirely?